tests/shell/pipeline/commands/internal.rs

```diff
@@ -23,8 +23,7 @@ fn takes_rows_of_nu_value_strings_and_pipes_it_to_stdin_of_external() {
         r#"
             open nu_times.csv
             | get name
-            | ^echo $it
-            | nu --testbin chop
```

Comment:
This was failing consistently for me locally, and perhaps is the reason behind: https://github.com/nushell/nushell/issues/1783.

Basically, `pipeline | <external>` will produce a single string, so `nth 3` was attempting to pull out the 3rd item in a pipeline emitting just a single string.

Note: I also tried `... | lines | nth 3 | echo $it`, but for some reason `lines` wasn't breaking the string up into lines. Will need to investigate that separately.

Reply:
Actually, we need `lines` to work, otherwise we're not properly testing stdin...